source/shared/cpp/ObjectModel/SharedAdaptiveCard.h

```diff
@@ -123,6 +141,14 @@ class AdaptiveCard
     std::vector<RemoteResourceInformation> GetResourceInformation();
 
     CardElementType GetElementType() const;
+    
+    std::unordered_map<std::string, AdaptiveCards::SemanticVersion>& GetRootRequires();
+    const std::unordered_map<std::string, AdaptiveCards::SemanticVersion>& GetRootRequires() const;
```

Comment:
@GabrielMedAlv Do we need both GetRootRequires() ?

Reply:
I put both methods more to conform similar structure as other functions in the same class like:

`std::vector<std::shared_ptr<BaseCardElement>>& GetBody();
    const std::vector<std::shared_ptr<BaseCardElement>>& GetBody() const;
    std::vector<std::shared_ptr<BaseActionElement>>& GetActions();
    const std::vector<std::shared_ptr<BaseActionElement>>& GetActions() const;`